etcdctl/README.md

```diff
@@ -565,6 +565,10 @@ Prints a humanized table of the member IDs, statuses, names, peer addresses, and
 
 ENDPOINT provides commands for querying individual endpoints.
 
+#### Options
+
+- cluster -- fetch and use all endpoints from the etcd cluster
```

Comment:
this also suffers from the same issue as the previous `auto-sync` option. if users have a proxy (not necessarily etcd proxy or etcd gPRC proxy) in front of etcd, and they can only access the cluster though the proxy, then cluster option will try to bypass the proxy. this is not the desired behavior. we probably need to warn about that here?